app/extensions.js

```diff
@@ -197,6 +198,13 @@ module.exports.init = () => {
   let enableExtensions = () => {
     installExtension(config.braveExtensionId, getExtensionsPath('brave'), {manifest_location: 'component', manifest: generateBraveManifest()})
 
+    if (getSetting(settings.PDFJS_ENABLED)) {
```

Comment:
can't you still use "save link as" if you want to download? I guess I'm fine either way